pages/linux/img2pdf.md

```diff
@@ -0,0 +1,32 @@
+# img2pdf
+
+> Command-line lossless converter of raster images to PDF
+> More information: <https://pypi.org/project/img2pdf/>.
```

Comment:
A link to the official repository is better
```suggestion
> More information: <https://gitlab.mister-muffin.de/josch/img2pdf>.
```

Reply:
I have commited your edits and will move this page to  `common/`. I think it works outside Linux, but haven't tested anything.